numba/cuda/compiler.py

```diff
@@ -249,8 +249,15 @@ def compile(self, args):
         Returns the `CompileResult`.
         """
         if args not in self._compileinfos:
+            # Repro for this fix: Issue #5311 reproducer
+            nvvm_options = {
+                'debug': self.debug,
+                # XXX TBC 'fastmath': fastmath, BUG!!! (test case seems to
```

Comment:
remove this, open issue

Reply:
Don't think this is an actual issue - compilation will use the nvvm options from the code library.